docs/pages/signer/solana-wallets/solana-signer-package.mdx

```diff
@@ -1,40 +1,59 @@
 ---
-title: Signing Package
+title: How to sign messages and send transactions on Solana
 description: How to sign messages and send transactions on Solana
 text: How to sign messages and send transactions on Solana
-link: /signer/solana-wallets/solana-signer-package
-slug: wallets/signer/solana-wallets/solana-signer-package
+link: /signer/solana-wallets/get-started
+slug: wallets/signer/solana-wallets/get-started
 ---
 
-# Create Signer
+If you're not using React, use the `@account-kit/signer` package to create smart wallets on Solana, sign messages, sign transactions, and sponsor transaction fees.
```

Comment:
This doesn't seem correct to me. Use the smart wallets, which we are using, on Solana. This is just Signer.